tests/lib/rules/no-useless-rename.js

```diff
@@ -239,6 +260,30 @@ ruleTester.run("no-useless-rename", rule, {
             errors: ["Destructuring assignment foo unnecessarily renamed.", "Destructuring assignment bar unnecessarily renamed."]
         },
         {
+            code: "const {foo: foo, ...stuff} = myObject;",
+            parserOptions: {
```

Comment:
My bad, I didn't realize the rule was fixable. I will add that in a few
hours, when I get back home. Thanks!
On May 30, 2016 3:34 PM, "Kai Cataldo" <notifications@github.com> wrote:

> In tests/lib/rules/no-useless-rename.js
> <https://github.com/eslint/eslint/pull/6288#discussion_r65104582>:
>
> > @@ -239,6 +260,30 @@ ruleTester.run("no-useless-rename", rule, {
> >              errors: ["Destructuring assignment foo unnecessarily renamed.", "Destructuring assignment bar unnecessarily renamed."]
> >          },
> >          {
> > +            code: "const {foo: foo, ...stuff} = myObject;",
> > +            parserOptions: {
>
> We should check the fixed output here as well - check the other invalid
> tests for the output key to see what this looks like
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/eslint/eslint/pull/6288/files/80d4c3c664cd314b393a4b31e2878a704683645a#r65104582>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AARWeoNObAlcbU4o3J-4iSuxw4-CESFLks5qG0m3gaJpZM4Ip_rq>
> .
>